Added nonfunctional profile picture edit and delete icons

sirQuail opened the pull request:
### Description
Adds nonfunctional edit and delete buttons only visible when hovering over a trainee profile picture. 
Will be expanded to include confirmation and upload modals and finally api implementation.

### Showcase
<img width="414" height="696" alt="image" src="https://github.com/user-attachments/assets/cd4b4902-c660-4780-b4d1-b4a02658645b" />

sirQuail committed:
Added profile picture edit and delete icons

stasel commented on client/src/components/ProfileSidebar.tsx:
```diff
@@ -37,8 +42,70 @@ export const ProfileSidebar = ({ traineeId }: ProfileSidebarProps) => {
+        justifyContent="center"
+        position="relative"
+      >
+        <Avatar
```
Maybe it is a good idea to start thinking about extracting the profile picture to it's own component so it will not clutter the sidebar component

sirQuail committed:
Implemented copilot suggestions
spearated profile picture modal for better readablity

sirQuail committed:
removed unused onHover line

stasel reviewed (approved):
Looks good

sirQuail committed:
Removed redundant ProfileSidebar file.
It is now found in profile-> components

sirQuail committed:
Resolved an issue where I added an older file:
said file did not have the seperate ProfilePictureModal

zalexa19 commented on client/src/features/trainee-profile/profile/ProfileSidebar.tsx:
```diff
@@ -18,76 +18,75 @@ interface ProfileSidebarProps {
  * @returns {ReactNode} A React element that renders profile page sidebar information and profile image.
  */
-const ProfileSidebar = ({ traineeId }: ProfileSidebarProps) => {
+export const ProfileSidebar = ({ traineeId }: ProfileSidebarProps) => {
```
This component also has a default export at the bottom of this file.
Could you please delete the default export?

zalexa19 commented on client/src/features/trainee-profile/profile/components/ProfilePictureModal.tsx:
```diff
@@ -0,0 +1,87 @@
+      position="relative"
+    >
+      <Avatar
+        src={data?.imageURL}
```
I'm not sure, can Avatar component handle undefined as a `src`?

zalexa19 reviewed (commented):
Hi! 
I looked at your pr, really happy that you extracted some code to a new component! 
I left a comment about passing the Trainee to this component, other than that it looks cool :)

sirQuail committed:
Implemented suggestions from zalexa19 code review
- removed default export from ProfilePictureModal and TraineeProfile components
- passed required props to ProfilePictureModal instead of the whole data object
